### ISSUE TITLE
An internal error occurred during: "reload maven project".

### ISSUE THREAD

**Issue by laeubi** [eclipse-m2e/m2e-core#199]
```
java.lang.NullPointerException
	at java.base/java.util.concurrent.ConcurrentHashMap$MapEntry.setValue(ConcurrentHashMap.java:3539)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:1429)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:1027)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator$ModelVisitor.visit(StringVisitorModelInterpolator.java:170)
	at org.apache.maven.model.interpolation.StringVisitorModelInterpolator.interpolateModel(StringVisitorModelInterpolator.java:107)
	at org.apache.maven.model.building.DefaultModelBuilder.interpolateModel(DefaultModelBuilder.java:789)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:393)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:448)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:414)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:377)
	at org.eclipse.m2e.core.internal.embedder.MavenImpl.readMavenProjects(MavenImpl.java:659)
	at org.eclipse.m2e.core.internal.project.registry.ProjectRegistryManager.lambda$13(ProjectRegistryManager.java:870)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:179)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:153)
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:101)
	at org.eclipse.m2e.core.internal.project.registry.ProjectRegistryManager.execute(ProjectRegistryManager.java:1027)
	at org.eclipse.m2e.core.internal.project.registry.ProjectRegistryManager.readProjectsWithDependencies(ProjectRegistryManager.java:866)
	at org.eclipse.m2e.core.internal.project.registry.ProjectRegistryManager.create(ProjectRegistryManager.java:210)
	at org.eclipse.m2e.core.internal.project.registry.MavenProjectManager.create(MavenProjectManager.java:81)
	at org.eclipse.m2e.core.ui.internal.actions.SelectionUtil.getMavenProject(SelectionUtil.java:259)
	at org.eclipse.m2e.editor.pom.MavenPomEditor.readMavenProject(MavenPomEditor.java:634)
	at org.eclipse.m2e.editor.pom.MavenPomEditor$3.run(MavenPomEditor.java:938)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

**Comment by vrubezhny** [comment]
This issue is already reported to Apache Maven: https://issues.apache.org/jira/browse/MNG-6819 and should be fixed in v.4.0.0

**Comment by mickaelistria** [comment]
This is expected to be fixed since Maven 3.8.2. As m2e now includes m2e 3.8.4, this should not be fixed. Please reopen if it's not hte case.
